src/app/bootstrap/bootstrap.page.ts

```diff
@@ -28,7 +33,25 @@ export class BootstrapPage implements OnInit {
   }
 
   async validateToken(jwt: string) {
-    await this.apiService.getList().subscribe(data => {
+    try {
+      await this.apiService.me("eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiI0IiwiaWF0IjoxNjA1MjU5NjEwLCJleHAiOjE2MDU4NjQ0MTB9.DgUJ5pg7AZZfw4aZXqrCxuBQlY-covLnG3XaFuOdcHVUDrqzLXgIM39UzSislzjs7w3Q3UNUWPT66rGNwQjquw").subscribe(data => {
+        console.log('data:', data);
+        localStorage.setItem('user', JSON.stringify(data));
+      });
+      await this.toastService.presentToast("Login successfull!");
+      this.router.navigateByUrl('/home');
+      
+      // await this.alertService.present(200);
+    } catch (e) {
+      // fixme when login page created (remove me)
+      this.router.navigateByUrl('/login');
```

Comment:
please create login page and home page then test these changes and then update pr

Reply:
we need to implement AUTH guard with interception like this article describe it

https://www.freakyjolly.com/ionic-angular-guards-tutorial/#.X66NU2gzaM8